### PR TITLE
TesForC01_10_ProductOfIntegers

### PR DESCRIPTION
**Pull request by DeedeeDL**
This program will create an unit testing for a previous program where we calculated the product between 2 numbers

### CODE REVIEW

File: src/main/java/org/example/C01_10_ProductOfIntegers.java [hunk 1]
```diff
@@ -3,13 +3,10 @@
...
 public class C01_10_ProductOfIntegers {
-    public static void main(String[] args) {
-        Scanner scanner = new Scanner(System.in);
-        System.out.println("Enter your first number: ");
-        int num1 = scanner.nextInt();
-        System.out.println("Enter your second number: ");
-        int num2 = scanner.nextInt();
-        System.out.println("The product of your numbers is: " + (num1 * num2));
-        
+    public static int prod (int num1, int num2){
```
Comment (andreiursudev) [inline src/main/java/org/example/C01_10_ProductOfIntegers.java]: Incorrect function name. Product is the result of multiplication.
Rename function to `multiply`.

Comment (andreiursudev) [inline src/main/java/org/example/C01_10_ProductOfIntegers.java]: Format code.

File: src/test/java/org/example/C01_10_ProductOfIntegersTest.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class C01_10_ProductOfIntegersTest {
+
+    @Test
+    public void prod() {
```
Comment (andreiursudev) [inline src/test/java/org/example/C01_10_ProductOfIntegersTest.java]: Rename test to `multiply`.

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/C01_10_ProductOfIntegers.java (hunk 1 above)
Comment (DeedeeDL) [reply src/main/java/org/example/C01_10_ProductOfIntegers.java]: done

File: src/test/java/org/example/C01_10_ProductOfIntegersTest.java (hunk 2 above)
Comment (DeedeeDL) [reply src/test/java/org/example/C01_10_ProductOfIntegersTest.java]: done

File: src/main/java/org/example/C01_10_ProductOfIntegers.java (hunk 1 above)
Comment (DeedeeDL) [reply src/main/java/org/example/C01_10_ProductOfIntegers.java]: done

**Review by andreiursudev — APPROVED** [review]

File: src/main/java/org/example/C01_10_MultiplyOfIntegers.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package org.example;
+
+public class C01_10_MultiplyOfIntegers {
+    public static int multiply(int num1, int num2) {
+        int multiply = num1 * num2;
+        return multiply;
+    }
+
```
Comment (andreiursudev) [inline src/main/java/org/example/C01_10_MultiplyOfIntegers.java]: Remove empty lines.

File: src/main/java/org/example/C01_10_MultiplyOfIntegers.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package org.example;
+
+public class C01_10_MultiplyOfIntegers {
+    public static int multiply(int num1, int num2) {
+        int multiply = num1 * num2;
+        return multiply;
```
Comment (andreiursudev) [inline src/main/java/org/example/C01_10_MultiplyOfIntegers.java]: Use correct English. Multiply is a verb. The result of multiplying is `multiplication` or `product`.

**Review by andreiursudev — CHANGES_REQUESTED** [review]

**Review by andreiursudev — APPROVED** [review]

**Pull request merged**